subprojects/docs/src/docs/userguide/build_init_plugin.adoc

```diff
@@ -104,7 +92,7 @@ The conversion process has the following features:
 
 
 [[sec:javaapplication_]]
-==== “`java-application`”
+=== “`java-application`”
```

Comment:
How about we drop all the `"` from the headers. So `java-application` instead of "`java-application`".

Reply:
Okay. I don't much care for the smart quotes here anyway.